src/pages/volunteer-page.html

```diff
@@ -246,7 +245,7 @@ <h3>You'll get:</h3>
 			</p>
 		</div>
 		<div class="info-block">
-			<iframe src="https://forms.gle/1wvgsR2wdcg84NZu5" width="640" height="2371" frameborder="0" marginheight="0" marginwidth="0">Loading...</iframe>
+			<iframe title="LibertyJS Volunteer Sign Up 2023" src="https://forms.gle/1wvgsR2wdcg84NZu5" width="640" height="2371" frameborder="0" marginheight="0" marginwidth="0">Loading...</iframe>
```

Comment:
Without a title, it's unclear what this iframe does to non-sighted users: https://dequeuniversity.com/tips/provide-iframe-titles